src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -247,3 +257,163 @@ def getStringRDD(self, varName, format):
         #    return rdd
         #except Py4JJavaError:
         #    traceback.print_exc()
+
+def getNumCols(numPyArr):
+	if len(numPyArr.shape) == 1:
+		return 1
+	else:
+		return numPyArr.shape[1]
+       
+def convertToJavaMatrix(sc, src):
```

Comment:
Looking at the return type, do you think `convertToMatrixBlock` might be more appropriate for the function name?